build_modules/lib/src/modules.dart

```diff
@@ -88,32 +87,15 @@ class Module {
       fromJson: _assetIdsFromJson)
   final Set<AssetId> directDependencies;
 
+  @JsonKey(name: 'm', nullable: true, defaultValue: false)
+  final bool isMissing;
```

Comment:
Let's add a comment why it would be missing.

Reply:
done